home-work-app/src/app/redux/reducers/movieEditor.ts

```diff
@@ -0,0 +1,25 @@
+import { MovieAction, MovieActionReducer, MovieState } from '../../types';
+
+const initialState: MovieState = {
+    movies: [],
+    isUpdated: false,
+    queryParams: {}
+};
+
+export const movieEditor: MovieActionReducer = (state: MovieState = initialState, action: MovieAction): MovieState => {
+    switch (action.type.toUpperCase()) {
+        case 'UPDATE':
```

Comment:
There should be at least sort movies, filter, movies and set movies action types.
Also you can add action for saving `isLoading` flag and later you will nedd to save search input value in store.